client/extensions/woocommerce/lib/generate-variations/test/index.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import generateVariations from '../index';
+
+describe( 'generateVariations', () => {
```

Comment:
Okay I've updated that test to add an additional color, and an additional option on the third type -- giving us 12 variations built on 3 options: Color - 3 Types, Size - 2 Types, Sleeves - 2 Types.